peachjam/templates/peachjam/saved_document/_update_modal.html

```diff
@@ -0,0 +1,81 @@
+{% load i18n %}
+<div hx-swap-oob="true"
+     id="saveDocumentModalDialog-{{ saved_document.document.id }}"
+     class="modal-dialog">
+  <div class="modal-content">
+    <div class="modal-header">
+      <p class="h5 modal-title">{% trans 'Document saved' %}</p>
+      <button type="button"
+              class="btn-close"
+              data-bs-dismiss="modal"
+              aria-label="Close"></button>
+    </div>
+    <div class="modal-body">
+      <form id="save-document-form-{{ saved_document.id }}"
+            hx-target="{{ target }}"
+            hx-post="{% url 'saved_document_update' saved_document.id %}?next={{ next }}">
+        {{ form.document }}
+        <p class="h5">{{ saved_document.document }}</p>
```

Comment:
This can be odd sometimes because CoreDocument.__str__ includes the document type. Either we change that (no reason not to), or we use `saved_document.document.title` here explicitly.